usb_ser_mon/usb_ser_mon.py

```diff
@@ -178,38 +183,38 @@ def usb_serial_mon(monitor, device, baud=115200, debug=False, echo=False):
                     serial_port.close()
                     return
                 if debug:
-                    for x in data:
-                        log.print("Serial.Read '%c' 0x%02x\r" % (x, ord(x)))
+                    for x in six.iterbytes(data):
+                        log.print("Serial.Read 0x%02x\r" % x)
```

Comment:
I'd prefer to keep the original (i.e. show the character and the hex value). You can use:
```
for x in data:
    log.print("Serial.Read '%c' 0x%02x\r" % (x, x))
```

Reply:
I'd prefer to keep it too, but it would work only if you stick to ascii. SInce data is a byte sequence, x could be a part of a multi-byte representation of a unicode character. Printing it on its own does not make much sense. I guess we could make it conditional  - use '%c' only if the output encoding does not support multi-byte characters, or if the user specified an option forcing this behaviour.